htlc/src/components/withdrawHtlc.js

```diff
@@ -0,0 +1,58 @@
+/* global AlgoSigner */
+import { LEDGER } from '../algosigner.config';
+import * as YAMLData from '../../artifacts/cache/htlc.py.yaml';
+
+const algosdk = require("algosdk");
+
+async function WithdrawHtlc() {
+  try {
+
+    const sndrAddr = prompt("Enter Escrow Address");
+    const recvAddr = prompt("Enter Receiver Address");
+    const secret = prompt("Enter Secret");
+    const secretBytes = new Uint8Array(Buffer.from(secret)); 
+
+    const txParams = await AlgoSigner.algod({
+      ledger: LEDGER,
+      path: `/v2/transactions/params`,
+    });
+
+    const lsig = algosdk.makeLogicSig(YAMLData.base64ToBytes, [secretBytes]);
```

Comment:
what do we get from `YAMLData.base64ToBytes` ?

Reply:
We get the `teal` code after the compilation.